packages/react/src/components/form/initial.tsx

```diff
@@ -87,7 +87,11 @@ const Oidc: React.FC<OidcProps> = ({ providers, handleClick }) => {
             icon={PROVIDER_TO_ICON[p.options?.provider]}
           >
             {text["initial.oidc"]}
-            <span className={styles.oidcProvider}>{p.options?.provider}</span>
+            <span className={styles.oidcProvider}>
+              {p.options?.provider === "azuread"
+                ? "Microsoft Account"
+                : p.options?.provider}
+            </span>
```

Comment:
I think we recently added provider names in the i18n table?

Reply:
Yep, in the dashboard :)